helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml

```diff
@@ -28,6 +28,7 @@ generic-service:
   # Environment variables to load into the deployment
   env:
     NODE_ENV: "production"
+    REDIS_ENABLED: "true"
```

Comment:
This is the only commit extra from the previous PR as we need this value to be able to deploy into DEV environment
@GurnankCheema @michalnawrockidj